app/templates/events/view/sessions/list.hbs

```diff
@@ -0,0 +1,90 @@
+<div class="sixteen wide column">
+  <table class="ui tablet stackable very basic table">
+    <thead>
+      <tr>
+        <th>{{ t 'State' }}</th>
+        <th>{{ t 'Title' }}</th>
+        <th>{{ t 'Speakers' }}</th>
+        <th>{{ t 'Track' }}</th>
+        <th>{{ t 'Short Abstract' }}</th>
```

Comment:
Please keep uniformity and follow the style guides. Somewhere you have used {{t ''}} whereas it is {{ t '' }}. Avoid spaces.